core/src/main/java/org/bitcoinj/coinjoin/CoinJoinAccept.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright 2022 Dash Core Group
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.bitcoinj.coinjoin;
+
+import org.bitcoinj.core.Message;
+import org.bitcoinj.core.NetworkParameters;
+import org.bitcoinj.core.ProtocolException;
+import org.bitcoinj.core.Transaction;
+import org.bitcoinj.core.Utils;
+
+import java.io.IOException;
+import java.io.OutputStream;
+
+// dsa
+public class CoinJoinAccept extends Message {
+    private int denomination;
```

Comment:
This may be a set of flags (0x1, 0x2, 0x4 mean different denominations), but we can add constants for it in a subsequent PR.  I don't remember how this is handled in Dash Core.

Reply:
Dash Core seems to have a vector of standard denominations: https://github.com/dashpay/dash/blob/develop/src/coinjoin/coinjoin.h#L366-L372